internal/checker/terraform/rules/aws/aws_backup_selection_ensure_efs_file_systems_are_added_in_backup_plans_of_aws_backup.rego

```diff
@@ -0,0 +1,66 @@
+# METADATA
+# title: "Be certain to include Elastic File System (Amazon EFS) file systems in the backup plans of AWS Backup"
+# description: "By establishing a backup strategy, you can guarantee that your data is consistently backed up and can be restored if data is lost or corrupted."
+# scope: package
+# related_resources:
+# - https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/backup_selection.html
+# custom:
+#   id: CB_TFAWS_351
+#   severity: LOW
+package lib.terraform.CB_TFAWS_351
+
+import future.keywords.if
+
+isvalid(block){
+	block.Type == "resource"
+    block.Labels[_] == "aws_backup_selection"
+}
+
+resource[resource] {
+    block := pass[_]
+	resource := concat(".", block.Labels)
+} 
+
+resource[resource] { 
+    block := fail[_]
+	resource := concat(".", block.Labels)
+} 
+
+getTheLabelForAwsEfsFileSystem[label]{
+    resource := input[_]
+    resource.Type == "resource"
+    resource.Labels[_] == "aws_efs_file_system"
+    label := concat(".", resource.Labels)
+}
+
+isValidResourceAttached{
+    resource := input[_]
+    resource.Type == "resource"
+    resource.Labels[_] == "aws_backup_selection"
+    resources := resource.Attributes.resources[_]
```

Comment:
Quick heads up about this sirs, this only seems to work if the element in the Terraform file has quotation marks. 

Like this: 

 resources = [
    "aws_efs_file_system.ok_efs.arn"
  ]